sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -1097,6 +1097,29 @@ def test_cosh_expansion():
     assert cosh(3*x).expand(trig=True).expand() == \
         3*sinh(x)**2*cosh(x) + cosh(x)**3
 
+def test_cosh_positive():
+    # See issue 11721
+    # cosh(x) is positive for real values of x
+    x = symbols('x')
+    k = symbols('k', real=True)
+    n = symbols('n', integer=True)
+
+    assert cosh(k).is_positive == True
```

Comment:
Use `is` rather than `==` in these tests. This should definitely give the singleton True and not anything else that happens to be equal to it.